gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -1041,6 +1046,11 @@ function PrepareRound()
         and GAMEMODE.force_plymodel ~= ""
     then
         GAMEMODE.playermodel = GAMEMODE.force_plymodel
+    elseif cvSelectUniqueModelPerRound:GetBool() then
+        local plys = player.GetAll()
```

Comment:
```suggestion
        local plys = select(2, player.Iterator())
```

This is a little bit faster and something that was added in the most recent gmod update

Reply:
well, nevermind, we reverted this for now. So don't apply this yet